src/main/java/Duke.java

```diff
@@ -1,10 +1,105 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
```

Comment:
I like how you made your code very easy to understand by having only functions in main!